fastfield_codecs/src/linear.rs

```diff
@@ -42,6 +43,28 @@ impl Column for LinearReader {
     }
 }
 
+impl OptionalColumn for LinearReader {
```

Comment:
Shouldn't we have a generic converter from Column to OptionalColuman?

struct MadeOptional<C>(C);

```rust
impl<T, C: Column<T>> OptionalColumn<T> for MadeOptional<C> {
}
```